packages/contracts-bedrock/scripts/DeployAuthSystem.s.sol

```diff
@@ -54,3 +54,34 @@ contract DeployAuthSystemInput is CommonBase {
         return _owners;
     }
 }
+
+contract DeployAuthSystemOutput is CommonBase {
+    Safe internal _safe;
```

Comment:
Similar to the above comment of:
```
// Note: these will need to be replaced with settings specific to the different Safes in the system.
```
I assume this also will get replaced with multiple safe addresses?

Reply:
Yes, there is a lot of work to be done on the actual deployment side, but for now I just want to get all the scaffolding in place, which will make it much easier to iterate on the system deployment piece afterwards.